blobfs/src/lib.rs

```diff
@@ -168,6 +169,20 @@ pub struct BlobFs {
 }
 
 impl BlobFs {
+    fn ensure_path_exist(path: &Path) -> io::Result<()> {
+        if path.as_os_str().is_empty() {
+            return Err(einval!("path is empty"));
+        }
+        if !path.exists() {
+            create_dir_all(path).map_err(|e| {
+                error!("create dir error: {:?}", path);
+                e
+            })?;
+        }
+
```

Comment:
If the path exists, it may not be a directory, should we throw an error?

Reply:
if the path is not a directory, it will throw an error in Blobfs mount function. So we just ensure path exist here.